src/message/MessageFull.js

```diff
@@ -75,20 +74,20 @@ class MessageFull extends PureComponent {
           />
           <View style={styles.contentWrapper}>
             <ScrollView style={styles.inner}>
-              <TouchableWithoutFeedback onLongPress={onLongPress}>
+              <TouchableOpacity onLongPress={onLongPress}>
```

Comment:
Yeah, I don't think it is good, we had TouchableOpacity before. Most apps I tested had no feedback on long press.